internal/awsservice/cloudwatchlogs.go

```diff
@@ -194,22 +152,15 @@ func ValidateLogsInOrder(t *testing.T, logGroup, logStream string, logLines []st
 
 		nextToken = output.NextForwardToken
 	}
-
-	// Validate that each of the logs are found, in order and in full.
-	assert.Len(t, foundLogs, len(logLines))
-	for i := 0; i < len(logLines); i++ {
-		expected := strings.ReplaceAll(logLines[i], "'", "\"")
-		actual := strings.ReplaceAll(foundLogs[i], "'", "\"")
-		assert.Equal(t, expected, actual)
-	}
+	return foundLogs, nil
 }
 
-// isLogGroupExists confirms whether the logGroupName exists or not
-func IsLogGroupExists(t *testing.T, logGroupName string) bool {
-
+// IsLogGroupExists confirms whether the logGroupName exists or not
+func IsLogGroupExists(logGroupName string) bool {
 	cwlClient, clientContext, err := getCloudWatchLogsClient()
```

Comment:
You can call the [cwlClient directly](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/internal/awsservice/constant.go#L38-L46) without using `getCloudWatchLogsClient`